tests/epyccel/test_array_as_func_args.py

```diff
@@ -0,0 +1,143 @@
+"""
+This test file is made for testing the functionality of passing numpy arrays
+in the function arguments.
+"""
+# pylint: disable=missing-function-docstring
+import numpy as np
+from numpy.random import randint, uniform
+
+from pyccel.epyccel import epyccel
+from pyccel.decorators import types
+
+int_types = ['int8', 'int16', 'int32', 'int64']
+float_types = ['float32', 'float64']
+
+def test_array_int_1d_scalar_add(language):
```

Comment:
why did you create new tests ?, can't you just use the one we use for fortran and mark the one that doen't work xfail for the c language

Reply:
no, I can't .

Reply:
the current tests are using some features not implemented yet in c.

Reply:
I think there are some tests at the end of test_arrays. Isn't there a fizz buzz one that should work?

Reply:
It looks like the existing tests that might work are in [loops.py](https://github.com/pyccel/pyccel/blob/master/tests/epyccel/modules/loops.py)